pkg/solana/config/config.go

```diff
@@ -9,41 +9,6 @@ import (
 	"github.com/smartcontractkit/chainlink-common/pkg/config"
 )
 
-// Global solana defaults.
-var defaultConfigSet = Chain{
```

Comment:
Initially I populated docs.toml from core, but then I noticed that these values were different, so I updated them. However, IIUC the core defaults were in effect, because core uses a cascading `SetFrom`, so I think they need to be switched back?
- [ ] switch docs.toml fields back to original core values